Server/mods/deathmatch/logic/CStaticFunctionDefinitions.cpp

```diff
@@ -4136,10 +4136,16 @@ bool CStaticFunctionDefinitions::WarpPedIntoVehicle(CPed* pPed, CVehicle* pVehic
     assert(pPed);
     assert(pVehicle);
 
+    if (uiSeat > pVehicle->GetMaxPassengers() ||
+        (uiSeat > 0 && pVehicle->GetMaxPassengers() == VEHICLE_PASSENGERS_UNDEFINED)
+    )
+        return false;
```

Comment:
I think 2 seperate if statements would be more readable.
```suggestion
    if (uiSeat > pVehicle->GetMaxPassengers())
        return false;

    if (uiSeat > 0 && pVehicle->GetMaxPassengers() == VEHICLE_PASSENGERS_UNDEFINED)
        return false
```